Week-1/catty-board/index.html

```diff
@@ -0,0 +1,130 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="/line-awesome/css/line-awesome.min.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="/style.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <nav>
+        <i class="la la-bars nav-item"></i>
+        <i class="la la-home nav-item"></i>
+        <i class="la la-search nav-item"></i>
+        <i class="la la-volume-up nav-item"></i>
+        <i class="la la-user nav-item"></i>
+        <i class="la la-spotify nav-item"></i>
+        <i class="la la-cog nav-item"></i>
+        <i class="la la-soundcloud nav-item"></i>
+      </nav>
+      <main>
+        <div class="info-card">
+          <img
+            src="https://vif1g.csb.app/src/assets/cattyboard.jpg"
+            alt="catty-board"
+            class="top-image"
+          />
+          <div class="info-card-info">
+            <section>
+              <h1>Catty Board Top 100 Single Charts (11.06.36)</h1>
+              <p>Unknown Artist</p>
+              <p>2016.Charts.100 songs</p>
+            </section>
+            <section class="info-card-buttons">
+              <button><i class="la la-play"></i>Play All</button>
+              <button><i class="la la-plus"></i>Add to</button>
+              <button><i class="la la-ellipsis-h"></i>More</button>
+            </section>
+          </div>
+        </div>
+        <!-- <div class="list-container">
+          <div class="card card-odd">
+            <p class="left-one">1.One Dance</p>
+            <p class="left-two">ASDFASDFASDF</p>
+            <p class="right-one">2:54</p>
+            <button class="right-two">CATTY CLOUD SYNC</button>
+          </div>
+          <div class="card card-even">
+            <p class="left-one">1.One Dance</p>
+            <p class="left-two">ASDFASDFASDF</p>
+            <p class="right-one">2:54</p>
+            <button class="right-two">CATTY CLOUD SYNC</button>
+          </div>
+          <div class="card card-odd">
+            <p class="left-one">1.One Dance</p>
+            <p class="left-two">ASDFASDFASDF</p>
+            <p class="right-one">2:54</p>
+            <button class="right-two">CATTY CLOUD SYNC</button>
+          </div>
+          <div class="card card-even">
+            <p class="left-one">1.One Dance</p>
+            <p class="left-two">ASDFASDFASDF</p>
+            <p class="right-one">2:54</p>
+            <button class="right-two">CATTY CLOUD SYNC</button>
+          </div>
+        </div> -->
+        <table>
+          <tr></tr>

```

Comment:
why blank <tr> tag? try to avoid blank tag.

Reply:
Resolved